### PR TITLE
Create service and dto for user

### PR DESCRIPTION
**Pull request by IamBeaver**
Create users service with 4 crud operations. Create dtos for user. Add http and users service to IoC.

### CODE REVIEW

File: src/dtos/UserDto.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import BaseUserDto from "./BaseUserDto";
```
Comment (anniemt6) [inline src/dtos/UserDto.ts:1]: Code Style.

File: src/dtos/BaseUserDto.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default interface BaseUserDto {
+    firstName: string,
+    lastName: string,
+    age: string
```
Comment (anniemt6) [inline src/dtos/BaseUserDto.ts]: Whong type of age.

File: src/services/UsersService.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { inject, injectable } from "inversify";
```
Comment (MarikNez) [inline src/services/UsersService.ts:1]: different quotes in import. You should follow only one qoute style through all project

File: src/services/UsersService.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    public async updateUser(user: UserDto): Promise<void> {
+        await this.httpService.send(`${controllerName}/UpdateUser?Id=${user.id}&FirstName=${user.firstName}&LastName=${user.lastName}&Age=${user.age}`, MethodType.PUT);
+    }
+
+    public async createUser(user: BaseUserDto): Promise<void> {
+        await this.httpService.send(`${controllerName}/CreateUser?FirstName=${user.firstName}&LastName=${user.lastName}&Age=${user.age}`, MethodType.POST);
+    }
+    
+}
```
Comment (MarikNez) [inline src/services/UsersService.ts]: Your httpservice.send return Response? How you handle response.data?

File: src/services/UsersService.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+@injectable()
+export default class DefaultUsersService implements UsersService {
+    public constructor(
+        @inject(ownTypes.httpService) private readonly httpService: HttpService 
+    ) {
+    }
+
+    public async getAllUsers(): Promise<UserDto[]> {
+        const result = await this.httpService.send<UserDto[]>(`${controllerName}/GetAllUsers`, MethodType.GET);
```
Comment (MarikNez) [inline src/services/UsersService.ts]: Extravariable

**Pull request merged**